io.github.endless_sky.endless_sky.appdata.xml

```diff
@@ -60,6 +60,28 @@
     </screenshot>
   </screenshots>
   <releases>
+    <release version="0.10.10" date="2024-10-26">
+      <description>
+        <p>This is a stable release, focused on fixing bugs and making some other small improvements.</p>
+        <ul>
+          <li>The biggest change this release is the removal of the "hold fire" command, added in 0.10.9, because of bugs associated with it that could not be adequately resolved prior to this release.</li>
```

Comment:
```suggestion
          <li>The biggest change in this release is the removal of the "hold fire" command, added in 0.10.9, because of bugs associated with it that could not be adequately resolved prior to this release.</li>
```

Reply:
I don't think there's anything wrong with "The biggest changes this release".